security/_index.md

```diff
@@ -4,4 +4,151 @@ summary: "Jakarta Security defines a standard for creating secure Jakarta EE app
 #<!--.................0123456789.123456789.123456789.123456789.123456789.123456789-->
```

Comment:
I'm curious @ivargrimstad. Do you know what this comment is doing?

I see it on a lot of the spec pages. Is the page automatically generated, and is this some artifact from that? Is the page safe to modify with the changes I'm introducing?